src/pages/EditReportFieldDropdown.tsx

```diff
@@ -67,21 +68,17 @@ function EditReportFieldDropdownPage({onSubmit, fieldKey, fieldValue, fieldOptio
                     text: fieldValue,
                 },
             ],
-
-            includeP2P: false,
-            canInviteUser: false,
-            includePolicyReportFieldOptions: true,
-            policyReportFieldOptions: validFieldOptions,
-            recentlyUsedPolicyReportFieldOptions: recentlyUsedOptions,
+            options: validFieldOptions,
+            recentlyUsedOptions,
```

Comment:
NAB - Should we re-order these to match the params better, and not have to name them? I could go either way

Reply:
sorry not sure what you mean, you mean that we rename `validFieldOptions` to `options` so we don't have to specify the object property - or to _not_ use an object for the `getReportFieldOptionsSection` but separate parameters?

I think we have no style guideline on that but as soon as a function takes more than three parameters I usually create an object type as single parameter. For me this helps with readability, understanding what other options you can pass without opening the function declaration (intelli sense) and avoid having to pass `undefined` as a parameter explicitly when when we want to skip passing a certain parameter

Reply:
yeah i think that's fine! I just meant that the order they're listed in the function definition is different than the order we're passing them here. But it's not a huge deal, no need to change